templates/projects/empty-server/files/server/middleware.development.json

```diff
@@ -0,0 +1,9 @@
+{
+  "final:after": {
+    "strong-error-handler": {
+      "params": {
+        "isDebugMode": true
```

Comment:
I believe the option is called `debug`, not `isDebugMode`.

@davidcheung could you please confirm?

@loay please add unit-tests to verify the behaviour. It's best to avoid modification of `process.env`, I think this may work well for you:

```js
var app = loopback({ localRegistry: true, loadBuiltinModels: true });
var bootOptions = {
  appRootDir: // path to scaffolded app
  env: 'development' // or 'production'
};
boot(app, bootOptions, function(err) {
  // TODO: handle error
  app.listen(0, function() {
    // make bad request to app.get('url') 
    // and check whether response contains stack trace
  });
});
```